velox/docs/functions/spark/json.rst

```diff
@@ -44,3 +44,14 @@ JSON Functions
         SELECT json_object_keys(''); -- NULL
         SELECT json_object_keys(1); -- NULL
         SELECT json_object_keys('"hello"'); -- NULL
+
+.. spark:function:: to_json(jsonObject) -> jsonString
+
+    Converts a Json object (ROW, ARRAY or MAP) into a JSON string.
```

Comment:
Please test your change with JsonFunctionsSuite and JsonExpressionsSuite, thanks.

Reply:
Test in https://github.com/apache/incubator-gluten/pull/9357.